apps/src/lib/kits/maker/PlaygroundConstants.js

```diff
@@ -40,6 +40,8 @@ export const SONG_CHARGE = [
   [null, 3/8], ['E4', 3/16], ['G4', 1]
 ];
 
+export const SONG_1D = ["B4", null, "B4", null, "G#4", "F#4", "E4", null, "E4"];
```

Comment:
It took me a minute to understand what `SONG_1D` was referring to here.  Is there a more descriptive name we could give this song?